main.py

```diff
@@ -1531,7 +1531,7 @@ async def gen_inventory(message, person_id):
         embedVar.add_field(name="None", value=f"u hav no cats {get_emoji('cat_cry')}", inline=True)
 
     if embedVar.description:
-        embedVar.description += f"\n{get_emoji('staring_cat')} Cats: {total}, Value: {round(valuenum)}"
+        embedVar.description += f"\n{get_emoji('staring_cat')} Cats: {"{:,}".format(total)}, Value: {"{:,}".format(round(valuenum))}"
```

Comment:
no f-strings? :megamind:
`f"{valuenum:,}"` I think

Reply:
indeed this is a better way